packages/docs/content/components/drawer.mdx

```diff
@@ -9,6 +9,15 @@ healthcare:
   sketchLink: GmlMnkL
 ---
 
+<ThemeContent onlyThemes={['medicare']}>
+
+### Guidance
```

Comment:
Let's move this down to the guidance section on the page that already exists and then we can also remove this duplicate header